src/boot/efi/stub.c

```diff
@@ -463,15 +527,28 @@ static EFI_STATUS load_addons(
 
                 if (devicetree_addons && PE_SECTION_VECTOR_IS_SET(sections + UNIFIED_SECTION_DTB)) {
                         *devicetree_addons = xrealloc(*devicetree_addons,
-                                                      *n_devicetree_addons * sizeof(DevicetreeAddon),
-                                                      (*n_devicetree_addons + 1) * sizeof(DevicetreeAddon));
+                                                      *n_devicetree_addons * sizeof(NamedAddon),
+                                                      (*n_devicetree_addons + 1) * sizeof(NamedAddon));
 
-                        (*devicetree_addons)[(*n_devicetree_addons)++] = (DevicetreeAddon) {
+                        (*devicetree_addons)[(*n_devicetree_addons)++] = (NamedAddon) {
                                 .blob = {
                                         .iov_base = xmemdup((const uint8_t*) loaded_addon->ImageBase + sections[UNIFIED_SECTION_DTB].memory_offset, sections[UNIFIED_SECTION_DTB].size),
                                         .iov_len = sections[UNIFIED_SECTION_DTB].size,
                                 },
-                                .filename =  xstrdup16(items[i]),
+                                .filename = xstrdup16(items[i]),
+                        };
+                }
+
+                if (ucode_addons && PE_SECTION_VECTOR_IS_SET(sections + UNIFIED_SECTION_UCODE)) {
+                        *ucode_addons = xrealloc(*ucode_addons,
+                                                 *n_ucode_addons * sizeof(NamedAddon),
+                                                 (*n_ucode_addons + 1)  * sizeof(NamedAddon));
```

Comment:
doubled space?

Reply:
You can drop the double spaces, it's just a typo from a previous PR